pkg/database/config.go

```diff
@@ -10,16 +10,25 @@ import (
 type PathType bool
 
 const (
-	DefaultPath PathType = false
-	TestPath    PathType = true
+	DefaultPath     PathType = false
+	TestPath        PathType = true
+	dockerBuildFlag          = "DOCKER_BUILD"
```

Comment:
This ENV is not currently set in the docker image, I'll add the variable to the docker compose file and another to the github actions script.